spec/factories/blog_posts.rb

```diff
@@ -22,7 +22,7 @@
 #
 FactoryBot.define do
   factory :blog_post do
-    blog_id { Faker::Number.number(digits: 4) }
+    blog_id { Faker::Number.unique.number(digits: 4) }
```

Comment:
when I need unique things I usually prefer going with `SecureRandom` module because the problem with Faker is that it can run out of trials and you get `Faker::UniqueGenerator::RetryLimitExceeded`

Reply:
Yeah, I'm aware of that issue with `Faker`. I didn't know I could use `SecureRandom` for this purpose, how should I use it?

Reply:
well now that I checked https://ruby-doc.org/stdlib-2.5.1/libdoc/securerandom/rdoc/SecureRandom.html I realized that there is no method to get just a random number with x amount of digits 😅 

other option would be using FactoryBot's `sequence` https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences . But it's up to you changing it or not

Reply:
Oh, right! I remember looking into that option before. I think it's the best option here. Thank you!